app/jobs/pair_request/auto_reject_job.rb

```diff
@@ -0,0 +1,12 @@
+class PairRequest
+  class AutoRejectJob < ApplicationJob
+    sidekiq_options queue: 'low'
+
+    # rubocop:disable Rails/SkipsModelValidations
+    def perform
+      PairRequest.where(status: :pending).where('pair_requests.when < ?',
```

Comment:
Very small and maybe not worth changing, but enums come with built-in scopes for each of the values. So `PairRequest.where(status: :pending)` can just be `PairRequest.pending`.

Reply:
great callout! (totally forgot about those)